app/model/Campaign.scala

```diff
@@ -32,7 +30,8 @@ case class Campaign(
   targets: Map[String, Long] = Map.empty
 ) {
 
-  def toItem = Item.fromJSON(Json.toJson(this).toString())
+  def toItem: Either[CampaignCentralApiError, Item] =
+    Option(Item.fromJSON(Json.toJson(this).toString())).map(Right(_)) getOrElse Left(JsonParsingError(""))
```

Comment:
This could be handled by Scanamo in the future? It returns `Either`s.

Reply:
That's just what I'm doing now.

Reply:
The synergy we've got going on here @kelvin-chappell!